Pod/Classes/ios/NYTPhotosOverlayView.m

```diff
@@ -49,6 +50,10 @@ - (void)layoutSubviews {
     }];
     
     [super layoutSubviews];
+
+    if ([self.captionView respondsToSelector:@selector(setPreferredMaxLayoutWidth:)]) {
```

Comment:
> If you're just interested in the caption view responding to `setPreferredMaxLayoutWidth:`, what's the point of the protocol?

So that I have something to cast to on the next line: `(id<NYTPhotoCaptionViewLayoutWidthHinting>)`